binding/src/main/java/se/fortnox/reactivewizard/binding/AutoBindModules.java

```diff
@@ -84,20 +84,13 @@ public void configure(Binder binder) {
         // Sort important due to overrides below
         Collections.sort(autoBindModules);
 
-        Module mergedModule = null;
+        Module mergedModule = bootstrapBindings;
 
         for (AutoBindModule m : autoBindModules) {
-            if (mergedModule == null) {
-                mergedModule = m;
-            } else {
-                mergedModule = Modules.override(mergedModule).with(m);
-            }
+            mergedModule = Modules.override(mergedModule).with(m);
             m.preBind();
         }
 
-        mergedModule = Modules.override(mergedModule)
```

Comment:
checked if the test injector uses this to defectively override modules?

Reply:
TestInjector should indeed override the modules. I made a special case for it.